src/main/java/com/jalasoft/selenium/daniel/jauregui/movies/Customer.java

```diff
@@ -9,12 +9,19 @@
 class Customer {
     private final String name;
     private final ArrayList<Rental> rentals = new ArrayList<Rental>();
+    private double totalAmount;
+    private int frequentRenterPoints;
+    private String statement;
+
     /**
      * Constructor of Customer class.
      * @param name : Name of the customer.
      */
     Customer(final String name) {
         this.name = name;
+        this.totalAmount = 0;
+        this.frequentRenterPoints = 0;
```

Comment:
I think this is not necessary.
you can have two methods that return the totalAmount and totalFrequentRenterPoints.
```java
public double calculateTotalAmount() {
    return \\something;
}

public int calculateTotalFrequentRenterPoints() {
    return \\something;
}
```